cmd/node/factory/structs.go

```diff
@@ -472,9 +473,14 @@ func CryptoComponentsFactory(args *cryptoComponentsFactoryArgs) (*Crypto, error)
 
 	txSignKeyGen := signing.NewKeyGenerator(ed25519.NewEd25519())
 
-	messageSignVerifier, err := systemVM.NewMessageSigVerifier(args.keyGen, singleSigner)
-	if err != nil {
-		return nil, err
+	var messageSignVerifier vm.MessageSignVerifier
```

Comment:
You can replace this line with: messageSignVerifier := &genesis.NilMessageSignVerifier{} and drop the else branch

Reply:
can't do it. as it says type missmatch